### PR TITLE
gitignore build output and vscode files

### PR DESCRIPTION
**Pull request by configurator**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,9 @@
 .dart_tool/
 .packages
 pubspec.lock
+
+# Build output
+build
+
+# IDE files
+.vscode
```
Comment (kevmoo) [inline .gitignore]: This should be in your personal `~/.gitignore_global` file - https://dart.dev/guides/libraries/private-files

Comment (configurator) [reply .gitignore]: Fair enough - do you agree that `build` should be here, or should I close the PR?

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,9 @@
 .dart_tool/
 .packages
 pubspec.lock
+
+# Build output
+build
```
Comment (kevmoo) [inline .gitignore]: Can't take PRs here, sadly. Tracking at https://github.com/flutter/flutter/issues/32330

**Comment by kevmoo** [comment]
Thanks! Will follow up with https://github.com/flutter/flutter/issues/32330

File: .gitignore (hunk 2 above)
Comment (configurator) [reply .gitignore]: I'll close the PR then.

**Pull request closed without merging**